distribution/lib/Standard/Base/0.0.0-dev/src/Nothing.enso

```diff
@@ -1,4 +1,5 @@
-from Standard.Base import Boolean, True
+import project.Data.Any.Any
```

Comment:
`Any` seems to not be used in this file. Why is it imported?

Reply:
It is used in the type signature of `if_nothing`.